src/config/QuoteList.ts

```diff
@@ -17,6 +17,13 @@ export const QuoteList: Quote[] = [
     text: "Believe you can and you’re halfway there.",
     author: "Theodore Roosevelt",
   },
+
+  { 
+    text: "Change is the end result of all true learning.",
+    author: "Leo Buscaglia",
+  },
+
+
```

Comment:
```suggestion
  { 
    text: "Change is the end result of all true learning.",
    author: "Leo Buscaglia",
  },
```
Let's go ahead and clean up the extra spacing here.